corehq/apps/app_manager/suite_xml/post_process/endpoints.py

```diff
@@ -61,13 +61,14 @@ def _make_session_endpoint(self, module, form=None):
             elif child.id in argument_ids:
                 self._add_datum_for_arg(frame, child.id)
 
-        return SessionEndpoint(
-            id=endpoint_id,
-            arguments=[Argument(id=i) for i in argument_ids],
-            stack=stack,
-        )
+        kwargs = {
+            "id": endpoint_id,
+            "arguments": [Argument(id=i) for i in argument_ids],
+            "stack": stack,
+        }
+        return SessionEndpoint(**kwargs)
```

Comment:
nit: this change seems unnecessary

Reply:
bfc6710f02615389d2e27a7e1bfc8438d66c778e